scripts.txt

```diff
@@ -0,0 +1,20 @@
+var h = prompt ('Podaj wysoko�c choinki');
+var star = '';
+
+if (h <= 0 ) {
```

Comment:
what's the difference between `var h` in line 1 and `h` in line 9?

Reply:
In line 1 is global.
In line 9 is "borrowed" from function.